src/reachability/maxSens.jl

```diff
@@ -40,53 +40,48 @@ end
 
 # This function is called by forward_network
 function forward_layer(solver::MaxSens, L::Layer, input::Hyperrectangle)
-    (W, b, act) = (L.weights, L.bias, L.activation)
-    center = zeros(size(W, 1))
-    gamma  = zeros(size(W, 1))
-    for j in 1:size(W, 1)
-        node = Node(W[j,:], b[j], act)
-        center[j], gamma[j] = forward_node(solver, node, input)
-    end
-    return Hyperrectangle(center, gamma)
-end
+    output = approximate_affine_map(L,  input)
```

Comment:
`output = overapproximate(AffineMap(L.weights, input, L.bias))` is the other option. Any preference on which is clearer for the paper @changliuliu?

Reply:
I like `approximate_affine_map`